src/main/java/org/jboss/xavier/analytics/pojo/input/workload/inventory/VMWorkloadInventoryModel.java

```diff
@@ -35,7 +35,9 @@
     private String product;
     private String version;
     private String host_name;
-    private boolean cpuAffinityNotNull;
+    private Boolean cpuAffinityNotNull;
```

Comment:
If now it's a class, and can be nullable..... would it be more appropiate to name the filed "cpuAffinity" ?

Reply:
Done